### ISSUE TITLE
Actionable feedback: Cloud Native Maturity Model

### ISSUE THREAD

**Issue by jimangel** [cncf/cartografos#48]
Hey all! @caniszczyk shared the [model](https://github.com/cncf/cartografos/issues/46) with the ex-CNCF-Ambassadors slack channel and I shared with with a few peers at Google. The following is the combined feedback:

- "A lot less meat there that I would have expected" (There was some disagreement here)
  - **TODO:** Add supporting links, content in sections, pictures, diagrams, or other content to break up repetition / smaller sections.
- On Level 4, Helm is discussed as guidance vs. recommendation, "You’ll be expecting most of your software to be packaged with Helm with the feedback loop being closed as quickly as possible to reduce configuration drift."
  - **TODO:** Re-word to be a recommendation vs. direct guidance to use helm (i.e. Kustomize and other tools could do the job). I think this is well done in the earlier sections, "you may be starting to write Helm Charts..." sounds a lot better than "you're using Helm by now."
- Kustomize is discussed at lower levels of maturity and then isn't mentioned again when some folks are leveraging it at scale.
  - **TODO:** I think generalizing the customization tooling in early stages or group all customization tooling together and mention the group throughout (maybe a glossary term?). Alternately, replace all tooling-by-name with a generic "automated configuration and deployment tooling" blurb.
  - This isn't a Kustomize vs. Helm problem, it's a perception of the right tool for the right job potentially being ignored. There's other generators too that are popular.
-  It makes sense to have the certificates from a sales point of view, but those could potentially be summarized at the beginning or end vs. sprinkled along the model.

**TL;DR:** Most of the feedback is around softening the tooling recommendations, otherwise it was well received.

**Comment by siforster** [comment]
Hi @jimangel,
Thank you for the feedback you provided.  We discussed this at our community meeting on the 24th of January and the group decided to separate out the three main areas you identified into separate issues so that we can effectively track them.
They are now: #49, #50, and #51.

Thank you for the effort you went to in circulating and collating this feedback - the group really appreciated it!

**Comment by jimangel** [comment]
Thanks @siforster! Feel free to close this issue when it makes sense for the team! Cheers!
